Dockerfile

```diff
@@ -1,38 +1,28 @@
 FROM node:10.13.0
 
-# Set up deploy user and working directory
-RUN adduser --disabled-password --gecos '' deploy
-RUN mkdir -p /app
+WORKDIR /app
 
-# Set up dumb-init
-ADD https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 /usr/local/bin/dumb-init
-RUN chown deploy:deploy /usr/local/bin/dumb-init
-RUN chmod +x /usr/local/bin/dumb-init
+# Set up deploy user and working directory
+RUN apt-get update && apt-get install \
+  dumb-init \
+  libgpm2 \
+  libslang2 \
+  mc \
+  mc-data \
+  unzip && \
+  adduser --disabled-password --gecos '' deploy
+
+# Install the packages
+COPY package.json yarn.lock ./
+RUN yarn install
 
-# Set up yarn
-RUN npm install -g yarn@1.9.4
-RUN chmod +x /usr/local/bin/yarn
+# Copy application code
+COPY . ./
 
-# Set up mc
-ADD https://dl.minio.io/client/mc/release/linux-amd64/mc /usr/local/bin/mc
```

Comment:
Maybe leave `mc` set up as it is used in https://github.com/artsy/metaphysics/blob/master/scripts/create-heapsnapshot.sh#L10